corehq/apps/app_manager/tests/data/list_apps/list_apps.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <apps>
-  <app profile="http://onering.com/a/rohan/apps/download/app_id/profile.xml?latest=true" domain="rohan" name="Untitled Application" environment="production" media-profile="http://onering.com/a/rohan/apps/download/app_id/media_profile.xml?latest=true"/>
```

Comment:
Hey, just curious why you decided to remove the `?latest=true` part? We never discussed which version of an app (latest saved vs. build vs. starred) it would be best for this to return, so there's definitely not a clear answer. I was envisioning that in a later iteration of this I would allow users to configure that anyway, so not a big deal for now. Does this mean that right now, this will return the latest starred version of the app?